test/Microsoft.AspNet.TestHost.Tests/TestServerTests.cs

```diff
@@ -56,53 +56,53 @@ public void ThrowsIfNoApplicationEnvironmentIsRegisteredWithTheProvider()
             Assert.Equal("RequestServices:True", result);
         }
 
-        [Fact]
-        public async Task CanChangeApplicationName()
-        {
-            var fallbackServices = CallContextServiceLocator.Locator.ServiceProvider;
-            var appName = "gobblegobble";
-
-            var builder = TestServer.CreateBuilder(fallbackServices, new Configuration(),
-                app =>
-                {
-                    app.Run(context =>
-                    {
-                        var appEnv = app.ApplicationServices.GetRequiredService<IApplicationEnvironment>();
-                        return context.Response.WriteAsync("AppName:" + appEnv.ApplicationName);
-                    });
-                },
-                configureServices: null);
-
-            builder.ApplicationName = appName;
-            var server = builder.Build();
-
-            string result = await server.CreateClient().GetStringAsync("/path");
-            Assert.Equal("AppName:" + appName, result);
-        }
-
-        [Fact]
-        public async Task CanChangeAppPath()
-        {
-            var fallbackServices = CallContextServiceLocator.Locator.ServiceProvider;
-            var appPath = ".";
-
-            var builder = TestServer.CreateBuilder(fallbackServices, new Configuration(),
-                app =>
-                {
-                    app.Run(context =>
-                    {
-                        var env = app.ApplicationServices.GetRequiredService<IApplicationEnvironment>();
-                        return context.Response.WriteAsync("AppPath:" + env.ApplicationBasePath);
-                    });
-                },
-                configureServices: null);
-
-            builder.ApplicationBasePath = appPath;
-            var server = builder.Build();
-
-            string result = await server.CreateClient().GetStringAsync("/path");
-            Assert.Equal("AppPath:" + appPath, result);
-        }
+        //[Fact]
```

Comment:
I used to have support on TestServer for overriding the ApplicationBasePath/ApplicationName, not sure we still need this with the config changes to allow setting the base path directly?  I ripped out the support for now and MVC functionals pass without this, so I think I can just nuke this once and for all